src/index.html

```diff
@@ -8,7 +8,26 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main" class="container-fluid loading-mask">
+      <div class="main-display-wrapper">
+        <h1 id="chartname"></h1>
+        <p class="date-banner">Updated:<span id="date-banner-data"></span></p>
+
+        <p class="disclmr" id="disclmr"></p>
+
+        <table class="table table-bordered">
+          <thead>
+            <tr class="dark-row">
+              <th scope="col">Currency</th>
+              <th scope="col">Description</th>
+              <th scope="col">Rate</th>
+              <th scope="col">Symbol</th>
+            </tr>
+          </thead>
+          <tbody id="price-data-table"></tbody>
+        </table>
+      </div>
+    </div>
```

Comment:
Good work to implement the HTML structure and the final product looks very close to the design. Well done.

One thing to consider is you have a lot of empty HTML tags before the Javascript executed. Maybe we only need an empty div and create all the DOMs in JS run time? You don't have anything to show before load from the JSON file anyway.

Reply:
Or you can create a "No Data" displayed table before data fetched